PennMobile/src/main/java/com/pennapps/labs/pennmobile/api/DiningRequest.java

```diff
@@ -0,0 +1,14 @@
+package com.pennapps.labs.pennmobile.api;
+
+import com.pennapps.labs.pennmobile.classes.Venue;
```

Comment:
Reformat imports. I think Intellij/Android Studio has a shortcut to do this, but I don't remember.